Frontend/src/app/menu/menu.component.html

```diff
@@ -1,4 +1,18 @@
 <h2>Menu</h2>
 <input class="menu-input-date" type="date" #menuInput>
-<button (click)="handleMenu(menuInput)">Show menu</button>
+<button (click)="handleMenu(menuInput) ">Show menu</button>
+
+
+@if (menuResponse) {
+  <h2>Menu:</h2>
+  <div *ngFor="let vorspeise of menuResponse.vorspeisen">
```

Comment:
bitte benutz @for und nicht ngFor
https://angular.dev/essentials/conditionals-and-loops#for-block